caffe2/CMakeLists.txt

```diff
@@ -829,6 +829,15 @@ endif()
 set_property(SOURCE ${ATen_CORE_SRCS} APPEND
     PROPERTY COMPILE_DEFINITIONS "TORCH_ASSERT_ONLY_METHOD_OPERATORS")
 
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+  if(USE_MPS)
+    enable_language(OBJC OBJCXX)
+    foreach(input_filename ${Caffe2_MPS_SRCS})
+      set_source_files_properties(${input_filename} PROPERTIES COMPILE_FLAGS "-DUSE_MPS=1")
+    endforeach(input_filename)
```

Comment:
Do we need this?

`USE_MPS` is conditioned on `MPS_FOUND` by default
https://github.com/pytorch/pytorch/blob/e9be0b0fb93c271ba3deb31b5ccc8f4c4f602ef9/CMakeLists.txt#L229-L231

and `-DUSE_MPS=1` flag is added here:

https://github.com/pytorch/pytorch/blob/e9be0b0fb93c271ba3deb31b5ccc8f4c4f602ef9/CMakeLists.txt#L971-L972

Reply:
I don't remember why I do that. I just remembered if I don't add DUSE_MPS there is no such argument during compilation. I think the new commit is good.